addon/components/date.js

```diff
@@ -0,0 +1,62 @@
+import Component from '@ember/component';
+import layout from '../templates/components/date';
+import moment from 'moment';
+
+const MONTHS = [
+  'January',
+  'February',
+  'March',
+  'April',
+  'May',
+  'June',
+  'July',
+  'August',
+  'September',
+  'October',
+  'November',
+  'December',
+];
+
+// TO DO: make this more flexible
+const YEARS = Array(...Array(100)).map((_, i) => `${i + 1940}`);
+
+export default Component.extend({
+  layout,
+  classNames: ['cs-component-date'],
+  attributeBindings: ['dataTestName:data-test-cs-component-date'],
+  dataTestName: true,
+  months: MONTHS,
+  years: YEARS,
+
+  changeSelected(ev) {
+    let value = String(ev.target.value);
```

Comment:
What's the purpose of using `String` here? Isn't `ev.target.value` already a string?